src/main/java/com/suse/salt/netapi/datatypes/target/CompoundMatcher.java

```diff
@@ -0,0 +1,37 @@
+package com.suse.salt.netapi.datatypes.target;
+
+/**
+ *
+ * Target for referencing compound matcher.
+ */
+public class CompoundMatcher implements Target<String> {
```

Comment:
Can compound matchers be used with Salt SSH? In this case you might want to also implement `SSHTarget<String>` here as for example in [`Glob`](https://github.com/SUSE/salt-netapi-client/blob/master/src/main/java/com/suse/salt/netapi/datatypes/target/Glob.java#L6).